src/main/java/data/exercises/ExerciseList.java

```diff
@@ -1,10 +1,14 @@
 package data.exercises;
 
+import textcolors.TextColor;
+import werkit.UI;
+
 import java.util.ArrayList;
 
 public class ExerciseList {
     private final ArrayList<String> exercisesList = new ArrayList<>();
     private int numberOfExercises = 0;
+    private final UI ui = new UI();
```

Comment:
Perhaps to be consistent with the rest of the program, this variable need not be `final`?